src/ontology/mondo-ingest.Makefile

```diff
@@ -172,20 +172,34 @@ update-jinja-sparql-queries: python-install-dependencies
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
-config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: config/%_exclusions.tsv component-download-%.owl $(REPORTDIR)/mirror_signature-%.tsv $(REPORTDIR)/component_signature-%.tsv metadata/%.yml python-install-dependencies
+#################
+### Exclusions ##
+#################
+# Exclusions: by ontology
+# TODO: add prereqs back
+# $(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv: config/%_exclusions.tsv component-download-%.owl $(REPORTDIR)/mirror_signature-%.tsv $(REPORTDIR)/component_signature-%.tsv metadata/%.yml python-install-dependencies
+$(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv:
 	python3 $(SCRIPTSDIR)/exclusion_term_expansion.py \
-	--onto-name $* \
-	--exclusions-path $(word 1,$^) \
-	--onto-path $(TMPDIR)/$(word 2,$^).owl \
-	--mirror-signature-path $(word 3,$^) \
-	--component-signature-path $(word 4,$^) \
-	--config-path $(word 5,$^)
-
-# Combines all tables
-config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
-	@rm config/term_exclusions.txt; rm config/exclusion_reasons.robot.template.tsv; \
-	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
-	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
+	--exclusions-path config/$*_exclusions.tsv \
```

Comment:
Made each arg more explicit. TBH, this was handy for me to do for development, because then if I wanted to, I could comment out the prerequisites and run it faster. Otherwise, It got stuck on missing `$(word N,$^)

Reply:
I do the same thing all the time as well.